app/src/main/java/com/duckduckgo/app/bookmarks/model/BookmarksRepository.kt

```diff
@@ -96,6 +100,10 @@ class BookmarksDataRepository(
         return bookmarksDao.getBookmarksByParentIdSync(parentId)
     }
 
+    override suspend fun getBookmarkFolderByParentId(parentId: Long): BookmarkFolder {
```

Comment:
Can you add a test to cover this method?

What will happen when parentId is not found or doesn't return any folder? Please also test that scenario.

Reply:
I've added the test.

In that case, the bookmark is treated as if it is stored in the default root folder under the name "Bookmarks".